lib/mix/tasks/ecto.load.ex

```diff
@@ -101,7 +101,7 @@ defmodule Mix.Tasks.Ecto.Load do
   end
 
   defp skip_safety_warnings? do
-    Mix.Project.config()[:start_permanent] != true
+    not Mix.Project.config()[:start_permanent]
```

Comment:
this will raise instead of returning false is start_permanent is not a boolean. i don't believe there is a reason to change this behaviour